Dungeons/src/main/java/dungeons/Room.java

```diff
@@ -0,0 +1,30 @@
+package dungeons;
+
+import dungeons.factory.RoomFactory;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import static dungeons.utils.Constants.*;
+
+public abstract class Room {
+    protected List<Room> adjacentRooms = new ArrayList<>();
```

Comment:
why initialize the list if maybe there are no adjacent rooms?

Reply:
removed the initialazation of the list.